Extractor/FrontendExtractor.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+namespace Bazinga\Bundle\JsTranslationBundle\Extractor;
+
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Component\Finder\Finder;
+use Symfony\Component\Translation\Extractor\AbstractFileExtractor;
+use Symfony\Component\Translation\MessageCatalogue;
+use Symfony\Component\Translation\Extractor\ExtractorInterface;
+
+final class FrontendExtractor extends AbstractFileExtractor implements ExtractorInterface
+{
+    const FUNCTION_STRING_ARGUMENT_REGEX = '\s?[\'"]([^"\'),]+)[\'"]\s?';
```

Comment:
this should use possessive quantifiers to avoid useless backtracking, allowing to match on bigger files.

Reply:
I'm not familiar with possessive quantifiers. My regex skills aren't strong enough.
Is this one correct ?

Reply:
yes (well, I would keep `[^"'),]++` for the inner of the string, not allowing empty ones.

and btw, why forbidding commas and parenthesis inside the string ?

Reply:
I'd say is to ensure the use of transalation key but the again I don't really know as I forked this feature where @jcchavezs left it.

Reply:
That is exactly the reason but if this can be improved that is great. What I'd request @stof is to drop here the cases we are trying to avoid so they can be added to unit tests.

Reply:
I see 2 issues here:

- forcing people to use translation keys to be able to use the extractor does not make sense. The extractor should not enforce such assumption.
- there is nothing saying that these chars are forbidden when using translation keys. Translation keys can still be any string (the only thing about them is that you don't consider them as being also the English message, and so you don't edit the key when fixing a typo in your English message)

Reply:
And here are a few cases:

```js
t.trans('some,thing')
t.trans('display(details)'
```